sale_cancel_reason/view/sale_order_cancel_reason_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_sale_order_cancel_reason_form">
```

Comment:
sale_order_cancel_reason_view.xml and sale_order_view.xml both files have similar code. Please remove any one of the files.